library/ssl_srv.c

```diff
@@ -4643,13 +4643,16 @@ static int ssl_parse_certificate_verify( mbedtls_ssl_context *ssl )
     }
 
     ret = mbedtls_pk_verify( peer_pk,
-                                      md_alg, hash_start, hashlen,
-                                      ssl->in_msg + i, sig_len );
+                             md_alg, hash_start, hashlen,
```

Comment:
@AndrzejKurek , why ther return code in this function is initialized this way?
https://github.com/ARMmbed/mbedtls/blob/7f656459f773bd445d1616e90ed420a5c9210973/library/ssl_srv.c#L4459

Do we want to add additional error code for verify function to prevent its' skipping ?

Reply:
Good question. I think that we can change it to the platform fault error and test if CI passes, I don't see a use of this value possible in the function flow.